### PR TITLE
add bitpos command

### LINKED ISSUE #105: redis 3.0 : BITPOS key bit [start] [end]
http://redis.io/commands/bitpos

### PR DESCRIPTION
**Pull request by gchudnov**
Added `bitpos` command, please review.

resolves #105

Thank you,
Grigoriy

### CODE REVIEW

File: src/test/scala/redis/commands/StringsSpec.scala [hunk 1]
```diff
@@ -57,6 +57,23 @@ class StringsSpec extends RedisStandaloneServer {
       Await.result(r, timeOut)
     }
 
+    "BITPOS" in {
+      val r = for {
+        s1 <- redis.set("bitposKey", "a+b") // 01100001 00101011 01100010
+        v1 <- redis.bitpos("bitposKey", 0)
+        v2 <- redis.bitpos("bitposKey", 1)
+        v3 <- redis.bitpos("bitposKey", 1, 1)
+        v4 <- redis.bitpos("bitposKey", 0, 3)
```
Comment (etaty) [inline src/test/scala/redis/commands/StringsSpec.scala]: could you add a test with 4 arguments?

Comment (gchudnov) [inline src/test/scala/redis/commands/StringsSpec.scala]: sure, my fault...

Comment (gchudnov) [inline src/test/scala/redis/commands/StringsSpec.scala]: Done! Please, take a look...

**Comment by etaty** [comment]
thanks

**Pull request merged**